proposals/assign-operator-overload.md

```diff
@@ -0,0 +1,440 @@
+# Assign operator overload
+
+* **Type**: Design proposal
+* **Authors**: Anže Sodja, Lóránt Pintér, Stefan Wolf, Roman Elizarov
+* **Status**: In review
+* **Discussion**: // TODO add KEEP issue
```

Comment:
Please, link to https://github.com/Kotlin/KEEP/issues/309 here.

Reply:
Done